exporter/debugexporter/debug_processor.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019, OpenCensus Authors
```

Comment:
nit: maybe rename this file debug_processor.go -> exporter.go or debug_expoter.go ?

Reply:
+1

Reply:
done.